yfinance/base.py

```diff
@@ -1751,6 +1751,9 @@ def get_mutualfund_holders(self, proxy=None, as_dict=False):
     def get_info(self, proxy=None) -> dict:
         self._quote.proxy = proxy or self.proxy
         data = self._quote.info
+        data['bookValuePerShare'] = data['bookValue']
+        bal_sheet = self.get_balance_sheet()
```

Comment:
I don't like this, introduces another GET. Most people fetching `info` don't want book value - let those that do directly request balance sheet.